bin/internal.js

```diff
@@ -9,8 +9,10 @@ const server = require('../lib/server/internal').create();
 
 logger.debug('config', config);
 db.ping().done(function() {
-  server.start(function() {
-    logger.info('listening', server.info.uri);
+  server.then((s) => {
+    s.start(function() {
```

Comment:
If we are changing lines, might be worthwhile to update to newer syntax?

Reply:
👍